packages/core/lib/src/core_widget_factory.dart

```diff
@@ -681,10 +681,18 @@ class WidgetFactory {
           ..[kCssDisplay] = kCssDisplayBlock
           ..tsb.enqueue(TextStyleOps.fontStyle, FontStyle.italic);
         break;
+      case 'div':
+        final displayExpressions = meta.element.styles.where((element) => element.property == 'display');
+        final displayExpression = displayExpressions.isNotEmpty ? displayExpressions.first : null;
+        final display = displayExpression?.term ?? 'block';
+        if (display == 'flex') {
+          meta.register(_flexOp(meta));
+        }
 
+        meta[kCssDisplay] = kCssDisplayBlock;
+        break;
```

Comment:
Flex is supported for all kind of tags, not just `DIV` right? I think we should handle flex for all of them in `parseStyleDisplay`.